src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -104,7 +111,7 @@ export const LongTextMenu: StoryObj<Args> = {
     viewport: {
       defaultViewport: 'tablet',
```

Comment:
Are these `defaultViewport` values not affected?

Reply:
Good call. we have 5 examples of this default viewport and a few of them aren't pointing to anything. Will update them to reference the keys from this viewport config